integration/tests/windows/windows_test.go

```diff
@@ -87,13 +87,14 @@ var _ = Describe("(Integration) [Windows Nodegroups]", func() {
 	}
 
 	Context("When creating a cluster with Windows nodegroups", func() {
-		DescribeTable("it should be able to run Windows pods", func(withOIDC bool, ami, workload string) {
-			createCluster(withOIDC, ami)
+		DescribeTable("it should be able to run Windows pods", func(withOIDC bool, ami, workload, containerRuntime string) {
+			createCluster(withOIDC, ami, containerRuntime)
 			runWindowsPod(workload)
 		},
-			Entry("windows when withOIDC is disabled", false, api.NodeImageFamilyWindowsServer2019FullContainer, "windows-server-iis.yaml"),
-			Entry("windows when withOIDC is enabled", true, api.NodeImageFamilyWindowsServer2019FullContainer, "windows-server-iis.yaml"),
-			Entry("windows 20H2", true, api.NodeImageFamilyWindowsServer20H2CoreContainer, "windows-server-iis-20H2.yaml"),
+			Entry("windows when withOIDC is disabled", false, api.NodeImageFamilyWindowsServer2019FullContainer, "windows-server-iis.yaml", api.ContainerRuntimeDockerForWindows),
+			Entry("windows when withOIDC is enabled", true, api.NodeImageFamilyWindowsServer2019FullContainer, "windows-server-iis.yaml", api.ContainerRuntimeDockerForWindows),
+			Entry("windows 20H2", true, api.NodeImageFamilyWindowsServer20H2CoreContainer, "windows-server-iis-20H2.yaml", api.ContainerRuntimeDockerForWindows),
+			Entry("windows with container runtime containerd", true, api.NodeImageFamilyWindowsServer20H2CoreContainer, "windows-server-iis-20H2.yaml", api.ContainerRuntimeContainerD),
```

Comment:
rather than a new test, perhaps just modify the two `withOIDC is disabled` so one has containerd, the other dockerd? or do you think that overloads it too much?

Reply:
Yeah, this whole thing is a bit dodgy since it creates 4 clusters. But honestly, I don't want to refactor it now.

It probably should be fine to create a single cluster then create multiple nodegroups instead of a whole cluster...

But since it's testing OIDC... Meh... I don't know.